js/src/components/SettingsReplacementVariableEditors.js

```diff
@@ -22,30 +24,51 @@ class SettingsReplacementVariableEditors extends React.Component {
 		super( props );
 	}
 
+	filterEditorSpecificReplaceVars( replacementVariables, pageType ) {
+		const {
+			editor_specific_replace_vars: editorSpecificReplaceVars = {},
```

Comment:
Wew, learned something new today. Thought this had to be the other way around. Neat! ;)